introduction.md

```diff
@@ -6,3 +6,7 @@ This is me ChibiBlasphem
 ## Code School favorites paths
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?